.gitignore

```diff
@@ -1,2 +1 @@
-*.pyc
-.DS_Store
```

Comment:
Should these be ignored? I think these are good to have in ignore, along with .swp, etc.

Reply:
Good catch, they must have been deleted by mistake.  They're restored now.